srcpkgs/libcap/template

```diff
@@ -10,19 +10,15 @@ make_use_env=yes
 short_desc="POSIX.1e capabilities suite"
 maintainer="skmpz <dem.procopiou@gmail.com>"
 license="GPL-2.0-only"
-homepage="http://sites.google.com/site/fullycapable/"
+homepage="https://sites.google.com/site/fullycapable/"
 changelog="https://sites.google.com/site/fullycapable/release-notes-for-libcap"
-distfiles="${KERNEL_SITE}/libs/security/linux-privs/libcap2/${pkgname}-${version}.tar.xz"
-checksum=5091b24247999fd7a5e62bd9ac8bc761cda29f9baa0d1a2ca6a46f13891b4f0f
+distfiles="${KERNEL_SITE}/libs/security/linux-privs/libcap2/libcap-${version}.tar.xz"
+checksum=15c40ededb3003d70a283fe587a36b7d19c8b3b554e33f86129c059a4bb466b2
 
 if [ "$CROSS_BUILD" ]; then
 	make_build_args="CROSS_COMPILE=${XBPS_CROSS_TRIPLET}-"
-fi
-
-if [ "$CHROOT_READY" ]; then
-	hostmakedepends="perl"
-	if [ -z "$CROSS_BUILD" ]; then
-		hostmakedepends+=" gperf"
+	if [ -z "$CHROOT_READY" ]; then
+		hostmakedepends="gperf"
 	fi
```

Comment:
Previously the logic was `if chroot ready and not cross build`, now it's `if cross build and not chroot ready`, this is not the same, is there a reason for this?

Reply:
No, it was simply a mistake. Should be fixed now.